docs/releases/2.2.0.rst

```diff
@@ -0,0 +1,21 @@
+================================================
+RapidSMS 2.2.0 release notes (under development)
```

Comment:
This file should be a template for the next release (2.1.0 is the current release). Please remove the release notes related to 2.1.0.